packages/backend-platform/src/WebsocketService.ts

```diff
@@ -0,0 +1,1348 @@
+import type { RestrictedMessenger } from '@metamask/base-controller';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why are we not using this variable? Should we use it instead of ignoring the fact that we are not using it?